app/controllers/races_controller.rb

```diff
@@ -132,15 +152,15 @@ def race_params
       :name, :date, :laps, :easy_laps, :description_url, :send_email,
       :registration_threshold, :categories, :email_body, :lock_race_results,
       :uci_display, :race_type, :pool_id, :league_id, :control_points_raw,
-      :picture_url, :location_url, :hidden, :started_at, :millis_display
+      :picture_url, :location_url, :hidden, :started_at, :millis_display,
+      :skip_auth, :description_text
     )
   end
 
   def check_race_result
     # TODO rijesi ovo groblje
```

Comment:
If you didn't translate this, it says "TODO resolve this shit" (not exactly but that's the meaning 😄 ). I even forgot why in the hell I added this so if you have time you might want to try to remove this since it has big performance impact.